doc/crm.8.adoc

```diff
@@ -3429,6 +3432,8 @@ instance attributes of that operation. A typical example is
 +OCF_CHECK_LEVEL+.
```

Comment:
How about two more suggestions:

1. 
from: "[[cmdhelp_configure_ms,define a master-slave resource]]"
to: "[[cmdhelp_configure_ms,define a master-slave resource (deprecated) ]]"

2.
from: "only once in group, clone, or master-slave objects."
to: "only once in group, or clone objects."

Reply:
Changed